activerecord/lib/active_record/relation/batches.rb

```diff
@@ -28,10 +28,11 @@ module Batches
     # ==== Options
     # * <tt>:batch_size</tt> - Specifies the size of the batch. Default to 1000.
     # * <tt>:start</tt> - Specifies the primary key value to start from.
+    # * <tt>:end_at</tt> - Specifies the ending point for the batch processing.
```

Comment:
What does an ending point mean? Let's clarify by mimicking the line above.

"Specifies the primary key value to end at."